tests/v2/actions/vai/vai.go

```diff
@@ -0,0 +1,21 @@
+package vai
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/vai"
+)
+
+func EnableVAI(client *rancher.Client, vaiEnabled *bool, once *sync.Once) error {
```

Comment:
can you add a godoc comment

Reply:
maybe also explicitly call this EnableVaiCaching ? This is a new feature to me so hard to know if there's a difference in enabling vai vs the vai cache. but since the error string says `vai caching` it is a little confusing